packages/devtools/lib/src/popup.dart

```diff
@@ -175,8 +175,14 @@ class PopupListView<T> implements CoreElementView {
     highlightedItem = null;
   }
 
-  void scrollAndHighlight(int row, int topPosition,
-      {bool top = false, bool bottom = false}) {
+  void scrollAndHighlight(
+    int row,
+    int topPosition, {
+    bool top = false,
+    bool bottom = false,
+  }) {
+    if (itemsAsList.isEmpty) return;
```

Comment:
odd we are trying to scroll in this case. The row index will be out of the range. Add a TODO to investigate why this method is called when the list is empty.